src/plugins/notebook/components/NotebookEntry.vue

```diff
@@ -272,16 +276,21 @@ export default {
                 return;
             }
 
-            const entryPos = this.entryPosById(entryId);
+            target.blur();
+
             const value = target.textContent.trim();
-            if (this.currentEntryValue !== value) {
-                target.textContent = value;
+            if (this.currentEntryValue === value) {
+                return;
+            }
 
-                const entries = getNotebookEntries(this.domainObject, this.selectedSection, this.selectedPage);
-                entries[entryPos].text = value;
+            target.textContent = value;
```

Comment:
Should we use Vue binding instead?

Reply:
tried but had some issues. May need to revisit in future.

Reply:
I have a few questions around this as well, and the general architecture of how entries are updated